lib/meadow_web/plugs/elasticsearch.ex

```diff
@@ -37,7 +37,13 @@ defmodule MeadowWeb.Plugs.Elasticsearch do
         Application.get_env(:meadow, Meadow.ElasticsearchCluster)
       )
 
-    case config |> config[:api].request(conn.method, path <> query_string, body, []) do
+    method = case conn.method do
+      x when is_atom(x) -> x
+      x when is_binary(x) -> String.to_atom(x)
```

Comment:
`String.to_atom` can get you into trouble with hitting the atom limit, I think it's a best practice to use `String.to_existing_atom` whenever possible instead (same goes for the next line).
```suggestion
      x when is_binary(x) -> String.to_existing_atom(x)
```

Reply:
`String.to_atom` will already reuse an existing atom if it’s been defined previously. The only difference is that `to_existing_atom` will throw an exception if the atom doesn’t already exist. I can’t see a situation in which that’s desirable in this case – we don’t have the option to proceed without an atom.

Reply:
Right on, thanks for the clarification!